### ISSUE TITLE
add a testimonial section to zio website

### ISSUE THREAD

**Issue by softinio** [zio/zio#3614]
ZIO is getting a lot of positive testimonials online without even asking for one. Be good to have them somewhere on our website in particular this one: 

Link to actual tweet: 

<https://twitter.com/odersky/status/1261966053514653702?s=20>

Screenshot in case: 

<https://shared.softinio.com/Screen-Shot-2020-05-17-14-23-49.04.png>

**Comment by ElaadF** [comment]
Hi ! I would like to try, i may need help to build the website locally :
I run `sbt docs/docusaurusCreateSite` but there is no CSS

**Comment by softinio** [comment]
@ElaadF Thanks for helping with this issue. 

From the project root do: 

1. `sbt docs/docusaurusCreateSite`
2. `cd website`
3. `yarn install`
4. `yarn start`

Once you do the last step it should open up the home page automatically in your web browser.

**Comment by hearnadam** [comment]
@khajavi perhaps this can be a part of the new website?

**Comment by khajavi** [comment]
@hearnadam Yeah! This section definitely deserves a place on ZIO's homepage.
